examples/fluids/qfunctions/newtonian.h

```diff
@@ -63,6 +63,94 @@ CEED_QFUNCTION(ICsNewtonianIG_Conserv)(void *ctx, CeedInt Q, const CeedScalar *c
   return ICsNewtonianIG(ctx, Q, in, out, STATEVAR_CONSERVATIVE);
 }
 
+// *****************************************************************************
+// This QFunction computes the L2 error.
+// *****************************************************************************
+CEED_QFUNCTION(Newtonian_L2Error)(void *ctx, CeedInt Q, const CeedScalar *const *in, CeedScalar *const *out) {
+  // Inputs
+  const CeedScalar(*q_data)[CEED_Q_VLA] = (const CeedScalar(*)[CEED_Q_VLA])in[0];
+  const CeedScalar(*q_true)[CEED_Q_VLA] = (const CeedScalar(*)[CEED_Q_VLA])in[1];
+  const CeedScalar(*q_soln)[CEED_Q_VLA] = (const CeedScalar(*)[CEED_Q_VLA])in[2];
+
+  // Outputs
+  CeedScalar(*q_error)[CEED_Q_VLA] = (CeedScalar(*)[CEED_Q_VLA])out[0];
+
+  // Quadrature Point Loop
+  CeedPragmaSIMD for (CeedInt i = 0; i < Q; i++) {
+    CeedScalar q_t[5] = {0}, q_s[5] = {0};
+    for (CeedInt j = 0; j < 5; j++) {
+      q_s[j] = q_soln[j][i];
+      q_t[j] = q_true[j][i];  // Automatically provided in the same variables as the solution.
+    }
+    for (CeedInt j = 0; j < 5; j++) q_error[j][i] = q_data[0][i] * (q_t[j] - q_s[j]) * (q_t[j] - q_s[j]);
```

Comment:
```suggestion
    for (CeedInt j = 0; j < 5; j++) q_error[j][i] = q_data[0][i] * Square(q_t[j] - q_s[j]);
```

Similar for the other functions as well (though if we combine them into a single helper function, then that becomes a mute point).